templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -55,14 +55,12 @@ spec:
   {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED" "false") }}
     anonymousAuth: false
     authorizationMode: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_MODE" "RBAC" }}
-    authorizationRbacSuperUser: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_RBAC_SUPER_USER" "admin" }}
   {{- end }}
   {{- if bool (getenv "KOPS_ADMISSION_CONTROL_ENABLED" "true") }}
     admissionControl:
     - NamespaceLifecycle
     - LimitRanger
     - ServiceAccount
-    - PersistentVolumeLabel
```

Comment:
Why are we dropping this?

Reply:
I think we want this so that we know which cluster volumes are associated with

Reply:
Deprecated since kubernetes 1.8 https://github.com/kubernetes/kubernetes/pull/52618

Reply:
wow, hard to stay on top of all that! =)